### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by vanthome** [noolsjs/nools#224]
The projects mentions in the README.md that is's abandoned but there seems to be some activity. Can you pls. clarify its status?

**Comment by doug-martin** [comment]
Yeah, we put that warning on the readme because we (c2fo) are not using it anymore, and I havent had any time to dedicate to the project. I moved it to its own org and did quite a bit of work [upgrading it to es6](https://github.com/doug-martin/nools/tree/es6) and havent had time to circle back around.

I'd love contributors and there have been some over the years but the barrier to entry is quite high. \

TLDR: I might come back to but no promises :(
